Automattic/woocommerce-payments
# Add E2E tests for merchant on-boarding

Fixes #7347 

#### Changes proposed in this Pull Request
- Add E2E tests for multi-currency merchant on-boarding.

#### Test Suites:
1. **Currency Selection and Management**: 
- Disable submit button when no currencies are selected.
- Allow multiple currencies to be selectable.
- Exclude already enabled currencies from selection.
- Display suggested currencies at the beginning of the list.
- Ensure selected currencies are enabled after form submission.
2. **Geolocation Features**: 
- Offer currency switch by geolocation.
- Preview currency switch by geolocation with USD and GBP.
3. **Currency Switcher Widget**: 
- Test currency switcher widget availability with Storefront theme.
- Assess widget availability with unsupported themes.

#### Testing instructions

1. Set up a clean E2E testing environment locally.
2. Run the command `npm run test:e2e merchant-admin-multi-currency-on-boarding`.
3. Check that the tests pass in your local setup.
6. Verify that all E2E tests pass in the GitHub (GH) checks.

-------------------


**Post merge**
